app/javascript/packs/elm.js

```diff
@@ -95,7 +95,7 @@ document.addEventListener("DOMContentLoaded", () => {
         resetIconWhite,
         tooltipIcon,
         heatMapThresholdValues,
-        isStreaming: data.isStreaming,
+        isActive: data.isActive,
```

Comment:
If you removed it from the flags on the elm side, then you can also delete this. (but as I wrote it might be needed on the elm side)